Containers/Deploy-.NET-Core-app-in-Windows-Containers-to-ECS/Readme.md

```diff
@@ -116,46 +116,50 @@ In this module we will configure a cluster for hosting our application using Ama
 
 ## Module 3: Create an ASP.NET Core Application
 
-In this module we will create a default ASP.NET Core MVC web application using the Microsoft-supplied project template.
+In this module we will create a default ASP.NET Core Web App using the Microsoft-supplied project template.
 
 - Time to Complete---10 mins
 
 ### Step 1: Create Application in Visual Studio for Windows
 
-If your development environment is Visual Studio 2017 or 2019 on Windows, you can create an ASP.NET Core application as follows:
+If your development environment is Visual Studio 2019 on Windows, you can create an ASP.NET Core application as follows:
 
 1. Open Visual Studio and then create a new project by navigating to the *File* \> *New* \> *Project* menu item.
 
-1. In the *New Project* dialog, select *Visual C\#* \> *.NET Core* \> *ASP.NET Core Web Application* template.
+1. In the *Create a New Project* dialog, select *ASP.NET Core Web App C#* template.
 
-1. Enter *TestAspNetCoreApp* in the **Name** field, and then click **Ok**.
+1. Enter *TestAspNetCoreApp* in the **Name** field, and then click **Next**.
 
-1. Choose the **Web Application (Model-View-Controller)** template, and then ensure **Enable Docker Support** is checked, and the **OS** drop-down is set to *Windows*. Click **Ok** to complete the wizard.
+1. Select the *Framework*(.NET 5.0 or .NET Core3.1)
+
+1. Select **Enable Docker** and the **OS** drop-down is set to *Windows*.
+
+1. Click **Create** to complete the wizard.
 
 The wizard creates all the files and configuration necessary for hosting an ASP.NET Core Web application in a container, although before deploying the solution, you'll need to edit the *Dockerfile* file to ensure it targets the same version of Windows Server used by the cluster.
 
-The Cluster Creation Wizard currently uses Windows Server 2016, so you'll need to locate and open the *Dockerfile* in Visual Studio and then update all references to *nanoserver-1709* to *nanoserver-sac2016*. Your file should be similar to that shown below:
+The Cluster Creation Wizard currently uses Windows Server version 1809, so you'll need to locate and open the *Dockerfile* in Visual Studio and then update all references of */dotnet/sdk:5.0* to */dotnet/aspnet:5.0-nanoserver-1809* or */dotnet/aspnet:3.1-nanoserver-1809* depending on the .NET version your application is running on. Your file should be similar to that shown below:
```

Comment:
"Version 1809" appears to be Windows Server 2019, OS Build 1809 (see https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info). Can we revise (if I'm correct)?

Reply:
Yes, you're correct. Changes done!